user-service/src/main/java/by/battle/userservice/entity/Role.java

```diff
@@ -0,0 +1,35 @@
+package by.battle.userservice.entity;
+
+import lombok.Data;
+import lombok.ToString;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@Entity
+@Table(name = "role")
+public class Role {
+
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(
+            name = "UUID",
+            strategy = "org.hibernate.id.UUIDGenerator"
+    )
+    private UUID id;
```

Comment:
as we talked.
i would prefer store this as String. while it's still be possible to use UUID generator